lib/tasks/award_badges.rake

```diff
@@ -0,0 +1,28 @@
+namespace :award_badges do
+
+# 
+  desc "award timekeeper, energetic and ambitious badges"
+  task award_timeline_badges: :environment do
```

Comment:
should we just call it `timeline_badges` as we have award in namespace?

Reply:
sure